types.go

```diff
@@ -0,0 +1,63 @@
+package fq
+
+import "time"
+
+// TODO(aaron-prindle) currently testing with one concurrent request
+const C = 1 // const C = 300
```

Comment:
I do not understand why we have C and G constants in addition to fields in FQScheduler; couldn't we just set the fields to constant values?

Reply:
I've updated this, this makes more sense, Queue is also using G (outside of FQScheduler) for updating the VirtualFinish time.  I've updated these values to be set in FQScheduler and made G an argument to Queue.VirtualFinish now.